[2024-08-08 08:01:27,598] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.load_heroes 2024-08-08T08:01:23.084125+00:00 [queued]>
[2024-08-08 08:01:27,614] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.load_heroes 2024-08-08T08:01:23.084125+00:00 [queued]>
[2024-08-08 08:01:27,614] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:01:27,615] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2024-08-08 08:01:27,615] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:01:27,632] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): load_heroes> on 2024-08-08T08:01:23.084125+00:00
[2024-08-08 08:01:27,637] {standard_task_runner.py:52} INFO - Started process 59 to run task
[2024-08-08 08:01:27,641] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dota_dag', 'load_heroes', '2024-08-08T08:01:23.084125+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dota.py', '--cfg-path', '/tmp/tmpey7fsww_', '--error-file', '/tmp/tmpxfsqr5vt']
[2024-08-08 08:01:27,642] {standard_task_runner.py:77} INFO - Job 8: Subtask load_heroes
[2024-08-08 08:01:27,703] {logging_mixin.py:104} INFO - Running <TaskInstance: dota_dag.load_heroes 2024-08-08T08:01:23.084125+00:00 [running]> on host ec67c4f7e31a
[2024-08-08 08:01:27,750] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dota_dag
AIRFLOW_CTX_TASK_ID=load_heroes
AIRFLOW_CTX_EXECUTION_DATE=2024-08-08T08:01:23.084125+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-08T08:01:23.084125+00:00
[2024-08-08 08:01:27,751] {python.py:118} INFO - Done. Returned value was: None
[2024-08-08 08:01:27,764] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=dota_dag, task_id=load_heroes, execution_date=20240808T080123, start_date=20240808T080127, end_date=20240808T080127
[2024-08-08 08:01:27,807] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-08 08:01:27,853] {local_task_job.py:146} INFO - Task exited with return code 0
